packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -1251,3 +1252,9 @@ function IsInScope(context: TCGCContext, entity: Operation): boolean {
   }
   return true;
 }
+
+export const $emptyStringAsNull: EmptyStringAsNullDecorator = (
+  context: DecoratorContext,
+  target: Model,
```

Comment:
why the target is model? i suppose it should apply to model property.

Reply:
change decorator's target to model property.